app/renderer/components/Session/Session.js

```diff
@@ -53,7 +53,7 @@ const Session = (props) => {
   };
 
   const loadNewSession = async (caps, attachSessId = null) => {
-    if (await newSession(caps, attachSessId)) {
+    if (await newSession(cloneDeep(caps), attachSessId)) {
```

Comment:
It's probably simpler to use `_.cloneDeep` and leave the import statement unchanged.